.github/workflows/rancher-components.yaml

```diff
@@ -0,0 +1,25 @@
+name: rancher-components
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '14.x'
+      
+      - name: Install packages
+        run: yarn install
+
+      - name: Verify build
+        run: yarn build:lib
+      
+      - name: Run linters
+        run: cd pkg/rancher-components && yarn lint
```

Comment:
Please add command specific for linting in the main repository, so we are able to run autofix with the flag `--fix`